napari/components/camera.py

```diff
@@ -0,0 +1,98 @@
+from ..utils.event import EmitterGroup
+
+
+class Camera:
+    """Camera object modeling position and view of the camera.
+
+    Parameters
+    ----------
+    center : 2-tuple or 3-tuple
+        Center point of camera view for 2D or 3D viewing.
+    scale : float
+        Zoom level.
```

Comment:
Can you expand on this a bit? In general, I find the `scale` parameters in `napari` to be confusing.

Reply:
yeah, this might need more attention, but have added something

Reply:
linking to #763 too